ui/imports/shared/views/SyncingDeviceView.qml

```diff
@@ -177,7 +177,7 @@ Item {
             clip: true
 
             model: SortFilterProxyModel {
```

Comment:
Ideally this filtering should be externalized, this view just rendering provided model with no extra logic like that here.

Reply:
Yeah I know, existing code; didn't want to touch it more than necessary